src/node_platform.cc

```diff
@@ -167,17 +167,21 @@ class WorkerThreadsTaskRunner::DelayedTaskScheduler {
 };
 
 WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int thread_pool_size) {
-  Mutex::ScopedLock lock(platform_workers_mutex_);
-  pending_platform_workers_ = thread_pool_size;
+  Mutex platform_workers_mutex;
+  ConditionVariable platform_workers_ready;
+  int pending_platform_workers;
```

Comment:
I guess you could also initialize the variable here? :)

Reply:
I also don't understand why the mutex needs to be taken before the number is set - but have no strong feelings about it.

Reply:
I've addressed this.

IMO, for casual readers of the source it should be *locally* obvious that `pending_platform_workers` is the data guarded by the lock.